### DISCUSSION TITLE
nordpool sensor unavailable

### DISCUSSION THREAD

**Discussion by trappify** [custom-components/nordpool#408] [Q&A] [answered]
Hi,
I've been using the nordpool component for several years, but now it stopped working for me. And I'm not sure if I messed it up somehow or if there is an outage? Tried to minimize my config to the following:

Home assistant core 2024.8.3
Component version 0.0.14

//
sensor:
  - platform: nordpool
    region: "SE3"
//

But it's still unavailable.

Anyone in SE3 where this still works?

**Comment by trappify** [comment, marked as answer]
Got it working now. Not sure exactly what I did, but I did a thorough cleanup of my configuration.yaml en restarted a few times. So probably just something wrong with my config structure. I noticed though that my previous sensor name was "sensor.nordpool_kwh_se3_sek_3_095_025" and with the minimal config it was "sensor.nordpool_kwh_se3_sek_3_10_025". Not sure what it means, but after going back to my original config, I got the old name back.

Old config (now working):
  - platform: nordpool
    region: "SE3"
    VAT: True
    currency: "SEK"
    low_price_cutoff: 0.95
    precision: 3
    price_type: kWh
    additional_costs: "{{0.0|float}}"

**Comment by bertgaster** [comment]
7-11-2024
Hi,

My Nordpool intergration stops working also. Restart did not help. I read somewhere that Norpool changed his API. Is that correct?

Log:
Logboek: Error during setup of component nordpool

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 288, in _async_setup_component
    result = await task
             ^^^^^^^^^^
  File "/config/custom_components/nordpool/__init__.py", line 197, in async_setup
    return await _dry_setup(hass, config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/nordpool/__init__.py", line 179, in _dry_setup
    tz=await dt_utils.async_get_time_zone("Europe/Stockholm"),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'homeassistant.util.dt' has no attribute 'async_get_time_zone'

<end log>

See also: https://www.nordpoolgroup.com/en/trading/Operational-Message-List/2024/05/market-data-api-v1-eol-migration-to-v2-20240508075500/

=> We would like to remind you that support for Market Data API v1 will officially end on June 30.

=> What are the major changes in API v2?
One of the major changes you’ll notice is that we have consolidated multiple endpoints into one with API v2. You can find the mapping for the endpoints in the FAQ [https://developers.nordpoolgroup.com/reference/data-faq].

Is there a new HACS integration of Nordpool available ?
